docs/INSTALL.md

```diff
@@ -511,7 +511,7 @@ bearer credentials allowing for delegation, attenuation, and other cool
 features. You can learn more about them in Alex Akselrod's [writeup on
 Github](https://github.com/lightningnetwork/lnd/issues/20).
 
-Running `lnd` for the first time will by default generate the `admin.macaroon`,
+Running `lncli create` to create a wallet for the first time will by default generate the `admin.macaroon`,
```

Comment:
Since you can only run `lncli create` once, maybe we could remove the "for the first time".